### PR TITLE
Openstack Cheat Sheet

### PR DESCRIPTION
**Pull request by mohan08p**
What does your Instant Answer do?

Show all The OpenStack basic commands as cheat sheet.

What is the data source for your Instant Answer? (Provide a link if possible)

http://docs.openstack.org/user-guide/cli_cheat_sheet.html

Why did you choose this data source?

OpenStack is a cloud operating system that manages your complete infrastructure through data-center. So, it is very helpful and easy for them to apply the commands.

Are there any other alternative (better) data sources?

No

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)

DevOPs, SystemAdmins, Open source, Git users, Geeks, etc.

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?

Yes, https://duck.co/ideas/idea/6345/openstack-cheat-sheet

Are you having any problems? Do you need our help with anything?

No

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)
openstack_snapshot

### CODE REVIEW

**Comment by daxtheduck** [comment]
@mohan08p Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: share/goodie/cheat_sheets/json/mongodb.json [hunk 1]
```diff
@@ -0,0 +1,343 @@
+{
+    "id": "mongodb_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mongodb.json]: Whoops, this shouldn't be here. Please only add 1 new Instant Answer per Pull Request

File: share/goodie/cheat_sheets/json/openstack.json [hunk 2]
```diff
@@ -0,0 +1,230 @@
+{
+    "id": "openstack_cheat_sheet",
+    "name": "OpenStack",
+    "description": "Command Line Interface",
+    "metadata": {
+        "sourceName": "OpenStack - Cheat Sheet",
+        "sourceUrl": "http://docs.openstack.org/user-guide/cli_cheat_sheet.html"
+    },
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/openstack.json]: Please make sure you specify a template. In this case the `terminal` template is ideal

File: share/goodie/cheat_sheets/json/openstack.json [hunk 3]
```diff
@@ -0,0 +1,230 @@
+{
+    "id": "openstack_cheat_sheet",
+    "name": "OpenStack",
+    "description": "Command Line Interface",
+    "metadata": {
+        "sourceName": "OpenStack - Cheat Sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/openstack.json]: Let's change this to `OpenStack`

**Pull request closed without merging**